src/test/java/org/jenkinsci/plugins/docker/workflow/WithContainerStepTest.java

```diff
@@ -185,8 +185,8 @@ public class WithContainerStepTest {
                 WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "prj");
                 p.setDefinition(new CpsFlowDefinition(
                     "node {\n" +
-                    "  withDockerContainer('httpd:2.4.12') {\n" +
-                    "    sh \"sleep 5; ps -e -o pid,command | egrep '${pwd tmp: true}/durable-.+/script.sh' | fgrep -v grep | sort -n | tr -s ' ' | cut -d ' ' -f2 | xargs kill -9\"\n" +
+                    "  withDockerContainer('httpd:2.4.54-alpine') {\n" +
+                    "    sh \"set -e && sleep 5; ps -A -o pid,args | egrep '${pwd tmp: true}/durable-.+/script.sh' | fgrep -v grep | sort -n | tr -s ' ' | cut -d ' ' -f2 | xargs kill -9\"\n" +
```

Comment:
The old code was failing because `ps` was not in the container (only if you update past 2.4.35, `ps` is in 2.4.12 locally, so IDK what was happening in CI), so we ended up trying to parse "command not found: ps" and that results in a QEMU segfault. I added `set -e` to avoid that kind of confusion in the future.

Then, since `httpd:2.4.54` doesn't have `ps` in the first place, I switched to `2.4.54-alpine`, but its `ps` is from Busybox, and so only supports `-o args`, not `-o command`. I can't remember if changing `-e` to `-A` was necessary.